stdlib/public/core/Integers.swift.gyb

```diff
@@ -588,6 +588,63 @@ def assignmentOperatorComment(operator, fixedWidth):
   /// - Parameters:
   ///   - lhs: The value to divide.
   ///   - rhs: The value to divide `lhs` by. `rhs` must not be zero.
+""",
+        '&+': """\
+  /// Stores the sum of the two given values in the left-hand-size variable, discarding any overflow.
```

Comment:
Please wrap to 80 columns and follow the style of the documentation above; that is, "Adds the two values and stores..."